### ISSUE TITLE
Issue with...

### ISSUE THREAD

**Issue by LiangZhang2017** [ejolly/pymer4#110]
Can we use pymer4 to do the cross validation work?

**Comment by ejolly** [comment]
@LiangZhang2017 [n principle sure - you can do cross-validation with any supervised learning model and multi-level models fall under that category. But there is no native support for CV in `pymer4`. I would recommend using [scikit-learn](https://scikit-learn.org/stable/) and rolling your own `Estimator` that wraps `.fit` and `.predict`. 

Or you can try out the [sklearn-lmer library](https://github.com/transatlantic-comppsych/sklearn-lmer). But that's an independent project that I'm not involved in. 

Also in future it would great to reserve the 'bug' label for issues in the library rather than discussion questions.
